Gaijinizer patch/Map363.txt

```diff
@@ -238,10 +237,10 @@ I understand...#GT
 
 そしてその思惑通り、私達はモルモーの提案に乗った…
 > CONTEXT: Map363/events/1/pages/0/157/Dialogue/enemy
-You used her because \\C[2]you knew that \\N[1]
-would withdraw as soon as we felt your sign...\\C[0]
-And, as you might expect, we took Mormo's
-proposal...#TriketosPartial
+You used her because \\C[2]you knew we'd withdraw the
+moment any of us felt your presence...\\C[0]
+And as your expected, we'd go along with Mormo's
+proposal.#Triketos/Ron
 > END STRING
```

Comment:
Minor rewording for tenses:
```
You used her because \\C[2]you knew we'd withdraw the
moment any of us felt your presence...\\C[0]
And as you expected, we played along with Mormo's
proposal.
```